Gemfile

```diff
@@ -31,8 +31,6 @@ group :development, :test do
   gem "byebug"
   gem "fuubar"
   gem "govuk_test"
-  gem "jasmine"
```

Comment:
We should remove jasmine from the dependabot configuration: https://github.com/alphagov/service-manual-publisher/blob/849d2a77ecae7ea2357085c10b218a5d00e91828/.github/dependabot.yml#L24-L25

Reply:
👍